src/components/__tests__/SkeletonButton.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { renderIntoDocument, render, fireEvent } from 'react-testing-library'
+import SkeletonButton from '../SkeletonButton'
+
+it('renders without crashing', () => {
+  const { container } = render(
+    <SkeletonButton>Test SkeletonButton</SkeletonButton>,
+  )
+  expect(container.firstChild).toMatchSnapshot()
+})
+
+it('renders with custom width', () => {
+  const { container } = render(
+    <SkeletonButton width={360}>Custom Width SkeletonButton</SkeletonButton>,
+  )
+  expect(container.firstChild).toMatchSnapshot()
+})
+
+it('calls onClick when button is clicked', () => {
+  const handleClick = jest.fn()
+  const { getByText } = renderIntoDocument(
+    <SkeletonButton onClick={handleClick}>
+      Clickable SkeletonButton
+    </SkeletonButton>,
+  )
+  fireEvent(
+    getByText('Clickable SkeletonButton'),
+    new MouseEvent('click', {
+      bubbles: true,
+      cancelable: true,
+    }),
+  )
+
```

Comment:
Ok, this is super trivial, but do you we want to figure out where we do and do not add newlines in our tests?